rafx-framework/Cargo.toml

```diff
@@ -31,6 +31,9 @@ serde_bytes = "0.11"
 glam = { version = "0.13.1", features = [ "serde" ] }
 parking_lot = "0.11.1"
 slotmap = "1.0.2"
+downcast-rs = "1.2.0"
+dashmap = "4.0.2"
```

Comment:
dashmap makes a bucket per CPU core and some CPUs have lots of cores, so we should be highly selective about how/where we use it. How are we using it in this change?

Reply:
It's used in two places during extract.
1. There's some per-feature work that would generate a bunch of allocations if not reused frame-to-frame, so those are put into a `DashMap` indexed by the `RenderFeatureId`. This could be replaced, I think, by a `Vec` of `TrustCell`.
2. It's used to hold the temporary `Vec`s containing the entity / render object pairs after they've been categorized by feature & sorted. This is indexed by a persistent ID tied to the `RenderView`'s `ViewFrustum`. I'm not sure if there is a good way to replace this usage.

Reply:
Removed all uses in the first category.

Reply:
I'd lean toward using a plain mutex until a profiler shows that as being a bottleneck.